### PR TITLE
[EC-828] Access selector layout bugs

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Fix the following issues:

1. Input fields have slightly different height
2. Select focus ring is not consistent with standard form elements (eg. the icon button next to it)
3. Permission column static cells are misaligned with editable cells (select input padding offset)

## Code changes

Following fixes were used:
1. Multi-select uses a `min-height` which was 1px too big. Removing `min-height` completely breaks the component. I've updated the rule to match the standard select, but we should really look into fixing it properly with padding or something.
2. Problem was that the border was removed completely. I've re-added it but made it transparent.
2 Extra. I've added border border on hover that matches the icon button. This is a new feature that needs to be reviewed.
3. This is also an issue that we need to follow up in the future. Chrome adds extra padding to select elements and the only way to remove it is using `appearence: none`. Unfortunately Firefox does not do this, meaning that we have different behavior making alignment across browsers impossible without completely removing the built-in select styling, which is how I unfortunately had to solve this issue. Removing the built-in styling means removing the chevron, so I've hade to add a custom one. We now have 3 different select chevrons in the dialogs, which really isn't optimal (see access selector in storybook and screenshots)

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Comment by coroiu** [comment]
This might have CSP issues, need to look into it

**Review by shane-melton — APPROVED** [review]
This looks good for temporary fixes. I do think we need to create some tech debt tickets to make sure we don't forget to fix these more "permanently".

Also, we'll need to make a PR to update the CSP rules in the [Bitwarden Misc Repo](https://github.com/bitwarden/misc)

File: apps/web/webpack.config.js [hunk 1]
```diff
@@ -234,7 +234,7 @@ const devServer =
                   'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='
                   'sha256-JVRXyYPueLWdwGwY9m/7u4QlZ1xeQdqUj2t8OVIzZE4='
                   'sha256-or0p3LaHetJ4FRq+flVORVFFNsOjQGWrDvX8Jf7ACWg='
-                  'sha256-Oca9ZYU1dwNscIhdNV7tFBsr4oqagBhZx9/p4w8GOcg='
+                  'sha256-jvLh2uL2/Pq/gpvNJMaEL4C+TNhBeGadLIUyPcVRZvY='
```
[the remark this answers is not in the document]
Comment (shane-melton) [reply apps/web/webpack.config.js:237]: I opened a PR https://github.com/bitwarden/misc/pull/144

**Comment by coroiu** [comment]
@shane-melton 
> we need to create some tech debt tickets to make sure we don't forget to fix these more "permanently".

Done!
Select issue: https://bitwarden.atlassian.net/browse/CL-78
Multi-select issues: https://bitwarden.atlassian.net/browse/CL-75

**Pull request merged**